### DISCUSSION TITLE
Out of sync MQTT settings after upgrade from 1.0.1 build 9 to 1.1.2-beta.1

### DISCUSSION THREAD

**Discussion by erte1** [thanksmister/android-mqtt-alarm-panel#187] [Q&A] [answered]
As the title suggests, my MQTT settings are now out of sync after my version upgrade.   I am having trouble determining which MQTT topic setting aligns with the following:

topic from HA to play audio
topic from HA to set sun
topic from HA to set weather
topic from Android panel to provide device sensor info (e.g. Battery)

Will you please advise which Android panel MQTT setting aligns with each?

**Comment by thanksmister** [comment]
You can check the wiki for MQTT topics and commands.   I have not seen any issues with MQTT.

**Comment by erte1** [comment]
Still unclear.    It used to work in previous version and now not at all.

**Comment by erte1** [comment, marked as answer]
I was able to resolve the issue by removing uninstalling the app and reinstalling.  There must have been residual code that conflicted with the latest version.   

Also, the Remote Command topic is not correct in the wiki.  While it is true that remote command is used to monitor for the weather and sun posts and listens to alarmpanel/command', the configuration for this must just be 'alarmpanel'.   There must be embedded code that adds the /command.   Likewise for sending commands.
